### PR TITLE
Removed some redundant 'else's using early return

### PR DESCRIPTION
**Pull request by pradipta**

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@pradipta Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-boot&pullRequestId=22528)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-projects/spring-boot&pullRequestId=22528) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@pradipta Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-boot&pullRequestId=22528)!

File: spring-boot-project/spring-boot-tools/spring-boot-configuration-processor/src/json-shade/java/org/springframework/boot/configurationprocessor/json/JSONObject.java [hunk 1]
```diff
@@ -596,12 +596,10 @@ public String optString(String name, String fallback) {
 	 */
 	public JSONArray getJSONArray(String name) throws JSONException {
 		Object object = get(name);
-		if (object instanceof JSONArray) {
-			return (JSONArray) object;
-		}
-		else {
+		if (!(object instanceof JSONArray)) {
```
Comment (snicoll) [inline spring-boot-project/spring-boot-tools/spring-boot-configuration-processor/src/json-shade/java/org/springframework/boot/configurationprocessor/json/JSONObject.java:599]: This is a shaded version of a json library so we shouldn't modify it. Can you please revert all changes in this file?

File: spring-boot-project/spring-boot-tools/spring-boot-configuration-processor/src/json-shade/java/org/springframework/boot/configurationprocessor/json/JSONTokener.java [hunk 2]
```diff
@@ -549,15 +549,13 @@ public static int dehexchar(char hex) {
 		if (hex >= '0' && hex <= '9') {
 			return hex - '0';
 		}
-		else if (hex >= 'A' && hex <= 'F') {
+		if (hex >= 'A' && hex <= 'F') {
```
Comment (snicoll) [inline spring-boot-project/spring-boot-tools/spring-boot-configuration-processor/src/json-shade/java/org/springframework/boot/configurationprocessor/json/JSONTokener.java:552]: All changes in this file should also be removed.

**Comment by snicoll** [comment]
@pradipta thank you for making your first contribution to Spring Boot. I've reverted the changes to the shaded code [as I described in my review](https://github.com/spring-projects/spring-boot/pull/22528#pullrequestreview-460012475). I've also reverted a more complex if/else statement that offer better readability as is IMO. Thanks again!

**Pull request closed without merging**